bubblewrap.c

```diff
@@ -75,6 +75,30 @@ int opt_info_fd = -1;
 int opt_seccomp_fd = -1;
 char *opt_sandbox_hostname = NULL;
 
+#define CAP_TO_MASK_0(x) (1L << ((x) & 31))
+#define CAP_TO_MASK_1(x) CAP_TO_MASK_0(x - 32)
+
+#define ALLOWED_CAPS0_UNSHARE_USER (CAP_TO_MASK_0 (CAP_CHOWN) | CAP_TO_MASK_0(CAP_SETUID) | CAP_TO_MASK_0 (CAP_SETGID) | \
+                                    CAP_TO_MASK_0 (CAP_FOWNER) | CAP_TO_MASK_0 (CAP_DAC_OVERRIDE) | CAP_TO_MASK_0 (CAP_FSETID) | \
+                                    CAP_TO_MASK_0 (CAP_SYS_ADMIN) | CAP_TO_MASK_0 (CAP_SETFCAP))
+#define ALLOWED_CAPS1_UNSHARE_USER (0)
```

Comment:
this was the case when running as privileged.  I am going to drop completely this part since it is not used anymore now.  It will make the patch much smaller and easier to follow